### PR TITLE
Fix home cache handler.

### PR DESCRIPTION
**Pull request by oliverchang**
Flask methods require a return value. Also simplify the return value of
testcase list cache to be consistent.

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR

**Comment by oliverchang** [comment]
> LGTM

Please remember to check this on other handlers that you migrate in the future :)

**Comment by iamarshsingh** [comment]
> Please remember to check this on other handlers that you migrate in the future :)
yes I was also confused about the cron jobs return, will take care of this in future and thanks for the fix.

File: src/appengine/handlers/home.py [hunk 1]
```diff
@@ -152,3 +151,4 @@ def get(self):
     """GET handler_flask."""
     # pylint: disable=unexpected-keyword-arg
     _get_all_project_results(__memoize_force__=True)
+    return 'OK'
```
Comment (inferno-chromium) [inline src/appengine/handlers/home.py:154]: Should we define this in constants somewhere and use everywhere.

**Pull request merged**